firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/Driver/I2C/ExtendedI2CDevice.h

```diff
@@ -7,7 +7,7 @@
 #pragma once
 
 #include "I2CMux.h"
-#include "Pufferfish/HAL/HAL.h"
+#include "Pufferfish/HAL/STM32/HAL.h"
```

Comment:
Let's only include `Pufferfish/HAL/Interfaces/I2CDevice.h` - drivers should not be including or using anything from `HAL/STM32`, so this PR is a great opportunity to fix this problem in the old code here.